TestAssets/TestProjects/AppWithDependencyOnMalformedPackage/NuGet.Config

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <config>
+  </config>
+  <packageSources>
+    <clear />
+    <add key="local" value="nuget-feed" />
+    <add key="dotnet-core" value="https://www.myget.org/F/dotnet-core/api/v3/index.json" />
+    <add key="cli-deps" value="https://dotnet.myget.org/F/cli-deps/api/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
+</configuration>
```

Comment:
It's because the test involves using a malformed Nuget package that has a dll without any metadata.

Does `dotnet restore --source "./nuget-feed"` achieve the same thing?